server/controllers/removePrisoner/removePrisoner.ts

```diff
@@ -0,0 +1,69 @@
+import type express from 'express'
+import { FormWizard } from 'hmpo-form-wizard'
+
+import { BaseController } from '../index'
+import { ReportWithDetails } from '../../data/incidentReportingApi'
+
+export default class RemovePrisoner extends BaseController {
+  middlewareLocals() {
+    super.middlewareLocals()
+  }
+
+  getBackLink(_req: FormWizard.Request, res: express.Response): string {
+    const reportId = res.locals.report.id
+    return `/reports/${reportId}/prisoners`
+  }
+
+  locals(req: FormWizard.Request, res: express.Response) {
+    const locals = super.locals(req, res)
+    const { errors } = res.locals
+    const report = res.locals.report as ReportWithDetails
+    const { index } = req.params
+
+    if (errors.removePrisoner) {
+      errors.removePrisoner.message = 'Select if you would like to remove this prisoner to continue.'
+    }
+
+    const prisonerToRemove = report.prisonersInvolved[Number(index) - 1]
```

Comment:
it’d be better to use `parseInt(index, 10)` here and in other places below: casting to a number is a bit too lenient